semcore/modal/src/Modal.jsx

```diff
@@ -13,17 +13,20 @@ import keyboardFocusEnhance from '@semcore/utils/lib/enhances/keyboardFocusEnhan
 import style from './style/modal.shadow.css';
 import { localizedMessages } from './translations/__intergalactic-dynamic-locales';
 import i18nEnhance from '@semcore/utils/lib/enhances/i18nEnhance';
+import { Text } from '@semcore/typography';
+import uniqueIDEnhancement from '@semcore/utils/lib/uniqueID';
 
 class ModalRoot extends Component {
   static displayName = 'Modal';
   static style = style;
-  static enhance = [i18nEnhance(localizedMessages)];
+  static enhance = [i18nEnhance(localizedMessages), uniqueIDEnhancement()];
   static defaultProps = {
     duration: 200,
     closable: true,
     i18n: localizedMessages,
     locale: 'en',
   };
+  state = { isTitle: false };
```

Comment:
`isTitle` name seems to be very confusing as far as oftenly title is a string entity representing a name of something. 
I suggest you to consider other naming like `hasTitle`